ambry-store/src/main/java/com/github/ambry/store/BlobStore.java

```diff
@@ -1227,6 +1245,14 @@ void compact(CompactionDetails details, byte[] bundleReadBuffer) throws IOExcept
     logger.info("One cycle of compaction is completed on the store {}", storeId);
   }
 
+  /**
+   * Closes the last log segment periodically if replica is in sealed status.
```

Comment:
Could you add comment describing how this period is determined?

Reply:
Added the comments. And later I'm planning to update this part since currently we are leveraging counter and timestamp to decided when should we running compactAllPolicy for hybridCompactionPolicy. Later we might speed up the frequency of compaction for main and video cluster, so counter frequency needs to be tuned. So the best way is only leveraging timestamp for the hybridCompactionPolicy.